explorer/interpreter/type_checker.cpp

```diff
@@ -2404,19 +2404,27 @@ static void RewriteMemberAccess(Nonnull<MemberAccessExpression*> access,
 }
 
 // Determine whether the given member declaration declares an instance member.
-static auto IsInstanceMember(Member member) {
-  if (!member.declaration()) {
-    // This is a struct field.
-    return true;
-  }
-  Nonnull<const Declaration*> declaration = *member.declaration();
-  switch (declaration->kind()) {
-    case DeclarationKind::FunctionDeclaration:
-      return cast<FunctionDeclaration>(declaration)->is_method();
-    case DeclarationKind::VariableDeclaration:
+static auto IsInstanceMember(Nonnull<const Member*> member) {
+  switch (member->kind()) {
+    case MemberKind::BaseClass:
+      [[fallthrough]];
+    case MemberKind::PositionalMember:
```

Comment:
```suggestion
    case MemberKind::BaseClass:
    case MemberKind::PositionalMember:
```

A `[[fallthrough]]` annotation isn't required if the case is empty.

Reply:
Done